lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -109,6 +109,10 @@ import Cardano.Wallet.Primitive.Types
     ( TokenBundleMaxSize (TokenBundleMaxSize) )
 import Cardano.Wallet.Primitive.Types.TokenBundle
     ( TokenBundle (..) )
+import Cardano.Wallet.Primitive.Types.TokenMap
+    ( AssetId )
+import Cardano.Wallet.Primitive.Types.TokenQuantity
+    ( TokenQuantity (..) )
```

Comment:
My preference would be to import these qualified as `W`, similar to `W.TxOut` and `W.TxIn`

Reply:
> My preference would be to import these qualified as `W`, similar to `W.TxOut` and `W.TxIn`

Thanks! I agree with you that we would benefit from using qualified imports here. I'll make a follow up PR after this one.